integrations/tensorflow/python_projects/iree_xla/setup.py

```diff
@@ -54,24 +60,25 @@ def finalize_options(self):
 
 
 setup(
-    name="iree-tools-xla@IREE_RELEASE_PACKAGE_SUFFIX@",
-    version="@IREE_RELEASE_VERSION@",
+    name="iree-tools-xla",
+    version="0.1",
     author="The IREE Team",
     author_email="iree-discuss@googlegroups.com",
+    license="Apache",
```

Comment:
```suggestion
    license="Apache-2.0 WITH LLVM-exception",
```

Reply:
These are all wrong and I will update en-masse. The last time I did this, PyPI complained because it has some validation.